src/mudlet.cpp

```diff
@@ -2555,105 +2555,105 @@ bool mudlet::closeWindow(Host* pHost, const QString& name)
     }
 }
 
-bool mudlet::setLabelClickCallback(Host* pHost, const QString& name, const QString& func, const TEvent& pA)
+bool mudlet::setLabelClickCallback(Host* pHost, const QString& name, const int func)
 {
     if (!pHost || !pHost->mpConsole) {
         return false;
     }
 
     auto pL = pHost->mpConsole->mLabelMap.value(name);
     if (pL) {
-        pL->setClick(func, pA);
+        pL->setClick(func);
         return true;
     } else {
         return false;
     }
 }
 
-bool mudlet::setLabelDoubleClickCallback(Host* pHost, const QString& name, const QString& func, const TEvent& pA)
+bool mudlet::setLabelDoubleClickCallback(Host* pHost, const QString& name, const int func)
 {
     if (!pHost || !pHost->mpConsole) {
         return false;
     }
 
     auto pL = pHost->mpConsole->mLabelMap.value(name);
     if (pL) {
-        pL->setDoubleClick(func, pA);
+        pL->setDoubleClick(func);
         return true;
     } else {
```

Comment:
:information_source: An anti-pattern that CLazy will pick up on and that we try now to eliminate - *else-after-return* - since the *if* will exit the function/method if it ***is*** true, there is no need for the `else` instead the following code (in this case the `return false;`) may be executed unconditionally outside of (and thus not indented one level) the `if` block...

... this is repeated down through all the other methods with the same functional layout and can be eliminated from all of them.